mmv1/products/logging/LinkedDataset.yaml

```diff
@@ -0,0 +1,130 @@
+  - !ruby/object:Api::Type::String
+    name: location
+    description: The location of the linked dataset.
+    url_param_only: true
+    immutable: true
+    default_from_api: true
+  - !ruby/object:Api::Type::String
+    # Make this a String for now since we don't have a good way to reference multiple resources.
+    name: bucket
+    description: The bucket to which the linked dataset is attached.
+    url_param_only: true
+    required: true
+    immutable: true
+    diff_suppress_func: 'compareResourceNames'
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    output: true
+    description: |
+      The resource name of the linked dataset. The name can have up to 100 characters. A valid link id
+      (at the end of the link name) must only have alphanumeric characters and underscores within it. 
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      Describes this link. The maximum length of the description is 8000 characters.
+  - !ruby/object:Api::Type::String
+    name: createTime
+    output: true
+    description: |
+      Output only. The creation timestamp of the link. A timestamp in RFC3339 UTC "Zulu" format,
+      with nanosecond resolution and up to nine fractional digits. Examples: "2014-10-02T15:01:23Z"
+      and "2014-10-02T15:01:23.045123456Z".
+  - !ruby/object:Api::Type::Enum
+    name: lifecycleState
+    output: true
+    description: |
+      Output only. The linked dataset lifecycle state.
+    values:
+      - :LIFECYCLE_STATE_UNSPECIFIED
+      - :ACTIVE
+      - :DELETE_REQUESTED
+      - :UPDATING
+      - :CREATING
+      - :FAILED
+  - !ruby/object:Api::Type::NestedObject
+    name: bigqueryDataset 
+    description: |
+      The information of a BigQuery Dataset. When a link is created, a BigQuery dataset is created along
+      with it, in the same project as the LogBucket it's linked to. This dataset will also have BigQuery
+      Views corresponding to the LogViews in the bucket.
+    default_from_api: true
+    output: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: datasetId 
+        output: true
+        description: |
+          Output only. The full resource name of the BigQuery dataset. The DATASET_ID will match the ID
+          of the link, so the link must match the naming restrictions of BigQuery datasets
+          (alphanumeric characters and underscores only). The dataset will have a resource path of
+          "bigquery.googleapis.com/projects/[PROJECT_ID]/datasets/[DATASET_ID]"
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "logging_linked_dataset_basic"
+    primary_resource_id: "logging_linked_dataset"
+    test_env_vars:
+      project: :PROJECT_NAME
+  - !ruby/object:Provider::Terraform::Examples
+    name: "logging_linked_dataset_all_params"
+    primary_resource_id: "logging_linked_dataset"
+    test_env_vars:
+      project: :PROJECT_NAME
```

Comment:
If you think this is a confusing pattern we can avoid emitting this test for the documentation (optional).

Reply:
I think it is useful for users to have both options (either full bucket link or parent+location+bucket name) -- slightly confusing potentially due to the API spec, but I think the location+project+bucket name method is consistent with how gcloud does it